v63003/u_inref/gtm8794.csh

```diff
@@ -37,8 +38,14 @@ $MUPIP FREEZE -OFF DEFAULT
 
 echo "# Updating Database"
 # Reassigning jnl file for the database to be in the temp folder (Randomly set to before/nobefore by the tests framework)
-$MUPIP SET -REGION DEFAULT -JOURNAL=disable -REPLICATION=off>&jnl.outx
-$MUPIP SET -REGION DEFAULT $tst_jnl_str -REPLICATION=on>&jnl.outx
+$MUPIP SET -REGION DEFAULT -JOURNAL=disable -REPLICATION=OFF>&jnl.outx
+if($status) then
+	echo "Bad MUPIP SET"
+endif
+$MUPIP SET -REGION DEFAULT $tst_jnl_str >&jnl.outx
```

Comment:
jnl.outx -> mupipset.out

We do not want two commands going to the same file without append (lines 41 and 45). One will overwrite the other's output.